allergies/allergies.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Importing Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--linking to the reset.css-->
+    <link rel="stylesheet" href="../css/reset.css">
+    <!--Importing materialize.css-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <!--linking to style.css-->
+    <link rel="stylesheet" href="../css/style.css">
+    <!--Lets browser know website is optimized for mobile-->
+    <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <!--title as it appears in the the tab-->
+    <title>REÇQUE</title>
+  </head>
+  <body>
+    <!--this represents the left side of the screen and also shows this section's grid layout for different screen sizes-->
+    <div class="row">
+      <div class="col m4 s12">
+        <div class="Mycontainer" >
+          <h3>Do you have any dietary restrictions or allergies?</h3>
+        </div>
+      </div>
+
+      <!--this represents the right side of the screen with the clickable icons and also shows this section's 
+        grid layout for different screen sizes. In this section we also have the set up for onclick events via 
+        id's and values for our functionality-->
+      <div class="col m8 s12">
+        <div class="Iconcontainer">
+          <div class="vl hide-on-small-only"> </div>
+          <div class="row">
+            <div value="peanut-free" class="col l3 m4 s4 notClicked healthIcon"><center><img id="peanut-freeIcon" class="responsive-img" src="../assets/white icons/peanut-free.png" style="width:58px; height:58px;"><p><strong>Peanut Free</strong></p></center></div>
```

Comment:
It's a best practice to avoid in-line styles. CSS should be in a CSS file or at least in a style tag.